pkg/test/framework/components/istio/operator.go

```diff
@@ -283,21 +272,39 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 		if errs := errG.Wait(); errs != nil {
 			return nil, fmt.Errorf("%d errors occurred deploying remote clusters: %v", errs.Len(), errs.ErrorOrNil())
 		}
-	}
 
-	if env.IsMultinetwork() {
-		// enable cross network traffic
-		for _, cluster := range env.KubeClusters {
-			if err := i.applyCrossNetworkGateway(cluster); err != nil {
+		if env.IsMulticluster() {
+			// For multicluster, configure direct access so each control plane can get endpoints from all
+			// API servers.
+			if err := i.configureDirectAPIServerAccess(ctx, env, cfg); err != nil {
```

Comment:
Could we just always do this, not just for multicluster? We want this to only apply the remote secret to `other` cluster anyway, so this should be a no-op for single cluster.

Generally, I'd like to get rid of the `if env.IsMulticluster()` blocks to the extent possible ... it shouldn't be a special case.

Reply:
removed

Reply:
Error patching Webhook Get "https://127.0.0.1:34611/apis/admissionregistration.k8s.io/v1beta1/mutatingwebhookconfigurations/istio-sidecar-injector": dial tcp 127.0.0.1:34611: connect: connection refused .   we can not create remote secret if it is only single cluster.. I turned it back.